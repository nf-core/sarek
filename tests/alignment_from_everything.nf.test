nextflow_pipeline {

    name "Test pipeline"
    script "../main.nf"
    tag "pipeline"
    tag "pipeline_sarek"

    test("-profile test --input bam_and_fastq_and_spring.csv --save_mapped --save_output_as_bam") {

        when {
            params {
                modules_testdata_base_path = 'https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/'
                igenomes_base   = 'https://raw.githubusercontent.com/nf-core/test-datasets/modules/data'
                input = "${projectDir}/tests/csv/3.0/bam_and_fastq_and_spring.csv"
                tools = ''
                outdir = "$outputDir"
                save_mapped = true
                save_output_as_bam = true
            }
        }

        then {
            // stable_name: All files + folders in ${params.outdir}/ with a stable name
            def stable_name = getAllFilesFromDir(params.outdir, relative: true, includeDir: true, ignore: ['pipeline_info/*.{html,json,txt}'])
            // stable_path: All files in ${params.outdir}/ with stable content
            def stable_path = getAllFilesFromDir(params.outdir, ignoreFile: 'tests/.nftignore')
            assertAll(
                { assert workflow.success},
                { assert snapshot(
                    // Number of successful tasks
                    workflow.trace.succeeded().size(),
                    // pipeline versions.yml file for multiqc from which Nextflow version is removed because we tests pipelines on multiple Nextflow versions
                    removeNextflowVersion("$outputDir/pipeline_info/nf_core_sarek_software_mqc_versions.yml"),
                    // All stable path name, with a relative path
                    stable_name,
                    // All files with stable contents
                    stable_path
                ).match() }
            )
        }
    }
}
