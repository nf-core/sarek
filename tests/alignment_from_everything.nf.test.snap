{
    "Run with profile test,alignment_from_everything | --save_mapped --save_output_as_bam": {
        "content": [
            27,
            {
                "BCFTOOLS_STATS": {
                    "bcftools": 1.2
                },
                "BWAMEM1_MEM": {
                    "bwa": "0.7.18-r1243-dirty",
                    "samtools": 1.2
                },
                "CRAM_TO_BAM": {
                    "samtools": 1.21
                },
                "CRAM_TO_BAM_RECAL": {
                    "samtools": 1.21
                },
                "FASTQC": {
                    "fastqc": "0.12.1"
                },
                "GATK4_APPLYBQSR": {
                    "gatk4": "4.5.0.0"
                },
                "GATK4_BASERECALIBRATOR": {
                    "gatk4": "4.5.0.0"
                },
                "GATK4_MARKDUPLICATES": {
                    "gatk4": "4.5.0.0",
                    "samtools": "1.19.2"
                },
                "INDEX_CRAM": {
                    "samtools": 1.21
                },
                "INDEX_MERGE_BAM": {
                    "samtools": 1.21
                },
                "MERGE_BAM": {
                    "samtools": 1.21
                },
                "MOSDEPTH": {
                    "mosdepth": "0.3.8"
                },
                "SAMTOOLS_STATS": {
                    "samtools": 1.21
                },
                "STRELKA_SINGLE": {
                    "strelka": "2.9.10"
                },
                "VCFTOOLS_TSTV_COUNT": {
                    "vcftools": "0.1.16"
                },
                "Workflow": {
                    "nf-core/sarek": "v3.6.0dev"
                }
            },
            [
                "csv",
                "csv/mapped.csv",
                "csv/markduplicates.csv",
                "csv/markduplicates_no_table.csv",
                "csv/recalibrated.csv",
                "csv/variantcalled.csv",
                "multiqc",
                "multiqc/multiqc_data",
                "multiqc/multiqc_data/bcftools_stats_indel-lengths.txt",
                "multiqc/multiqc_data/bcftools_stats_vqc_Count_Indels.txt",
                "multiqc/multiqc_data/bcftools_stats_vqc_Count_SNP.txt",
                "multiqc/multiqc_data/bcftools_stats_vqc_Count_Transitions.txt",
                "multiqc/multiqc_data/bcftools_stats_vqc_Count_Transversions.txt",
                "multiqc/multiqc_data/fastqc-status-check-heatmap.txt",
                "multiqc/multiqc_data/fastqc_adapter_content_plot.txt",
                "multiqc/multiqc_data/fastqc_per_base_n_content_plot.txt",
                "multiqc/multiqc_data/fastqc_per_base_sequence_quality_plot.txt",
                "multiqc/multiqc_data/fastqc_per_sequence_gc_content_plot_Counts.txt",
                "multiqc/multiqc_data/fastqc_per_sequence_gc_content_plot_Percentages.txt",
                "multiqc/multiqc_data/fastqc_per_sequence_quality_scores_plot.txt",
                "multiqc/multiqc_data/fastqc_sequence_counts_plot.txt",
                "multiqc/multiqc_data/fastqc_sequence_duplication_levels_plot.txt",
                "multiqc/multiqc_data/fastqc_sequence_length_distribution_plot.txt",
                "multiqc/multiqc_data/fastqc_top_overrepresented_sequences_table.txt",
                "multiqc/multiqc_data/gatk-base-recalibrator-quality-scores-plot_Pre-recalibration_Count.txt",
                "multiqc/multiqc_data/gatk-base-recalibrator-quality-scores-plot_Pre-recalibration_Percent.txt",
                "multiqc/multiqc_data/gatk-base-recalibrator-reported-empirical-plot.txt",
                "multiqc/multiqc_data/gatk_base_recalibrator.txt",
                "multiqc/multiqc_data/mosdepth-coverage-per-contig-single.txt",
                "multiqc/multiqc_data/mosdepth-cumcoverage-dist-id.txt",
                "multiqc/multiqc_data/mosdepth_cov_dist.txt",
                "multiqc/multiqc_data/mosdepth_cumcov_dist.txt",
                "multiqc/multiqc_data/mosdepth_perchrom.txt",
                "multiqc/multiqc_data/multiqc.log",
                "multiqc/multiqc_data/multiqc_bcftools_stats.txt",
                "multiqc/multiqc_data/multiqc_citations.txt",
                "multiqc/multiqc_data/multiqc_data.json",
                "multiqc/multiqc_data/multiqc_fastqc.txt",
                "multiqc/multiqc_data/multiqc_general_stats.txt",
                "multiqc/multiqc_data/multiqc_picard_dups.txt",
                "multiqc/multiqc_data/multiqc_samtools_stats.txt",
                "multiqc/multiqc_data/multiqc_software_versions.txt",
                "multiqc/multiqc_data/multiqc_sources.txt",
                "multiqc/multiqc_data/picard_deduplication.txt",
                "multiqc/multiqc_data/picard_histogram.txt",
                "multiqc/multiqc_data/picard_histogram_1.txt",
                "multiqc/multiqc_data/picard_histogram_2.txt",
                "multiqc/multiqc_data/samtools-stats-dp.txt",
                "multiqc/multiqc_data/samtools_alignment_plot.txt",
                "multiqc/multiqc_data/vcftools_tstv_by_count.txt",
                "multiqc/multiqc_data/vcftools_tstv_by_qual.txt",
                "multiqc/multiqc_plots",
                "multiqc/multiqc_plots/pdf",
                "multiqc/multiqc_plots/pdf/bcftools_stats_indel-lengths.pdf",
                "multiqc/multiqc_plots/pdf/bcftools_stats_vqc_Count_Indels.pdf",
                "multiqc/multiqc_plots/pdf/bcftools_stats_vqc_Count_SNP.pdf",
                "multiqc/multiqc_plots/pdf/bcftools_stats_vqc_Count_Transitions.pdf",
                "multiqc/multiqc_plots/pdf/bcftools_stats_vqc_Count_Transversions.pdf",
                "multiqc/multiqc_plots/pdf/fastqc-status-check-heatmap.pdf",
                "multiqc/multiqc_plots/pdf/fastqc_adapter_content_plot.pdf",
                "multiqc/multiqc_plots/pdf/fastqc_per_base_n_content_plot.pdf",
                "multiqc/multiqc_plots/pdf/fastqc_per_base_sequence_quality_plot.pdf",
                "multiqc/multiqc_plots/pdf/fastqc_per_sequence_gc_content_plot_Counts.pdf",
                "multiqc/multiqc_plots/pdf/fastqc_per_sequence_gc_content_plot_Percentages.pdf",
                "multiqc/multiqc_plots/pdf/fastqc_per_sequence_quality_scores_plot.pdf",
                "multiqc/multiqc_plots/pdf/fastqc_sequence_counts_plot-cnt.pdf",
                "multiqc/multiqc_plots/pdf/fastqc_sequence_counts_plot-pct.pdf",
                "multiqc/multiqc_plots/pdf/fastqc_sequence_duplication_levels_plot.pdf",
                "multiqc/multiqc_plots/pdf/fastqc_sequence_length_distribution_plot.pdf",
                "multiqc/multiqc_plots/pdf/fastqc_top_overrepresented_sequences_table.pdf",
                "multiqc/multiqc_plots/pdf/gatk-base-recalibrator-quality-scores-plot_Pre-recalibration_Count.pdf",
                "multiqc/multiqc_plots/pdf/gatk-base-recalibrator-quality-scores-plot_Pre-recalibration_Percent.pdf",
                "multiqc/multiqc_plots/pdf/gatk-base-recalibrator-reported-empirical-plot.pdf",
                "multiqc/multiqc_plots/pdf/general_stats_table.pdf",
                "multiqc/multiqc_plots/pdf/mosdepth-coverage-per-contig-single-cnt.pdf",
                "multiqc/multiqc_plots/pdf/mosdepth-coverage-per-contig-single-pct.pdf",
                "multiqc/multiqc_plots/pdf/mosdepth-cumcoverage-dist-id.pdf",
                "multiqc/multiqc_plots/pdf/picard_deduplication-cnt.pdf",
                "multiqc/multiqc_plots/pdf/picard_deduplication-pct.pdf",
                "multiqc/multiqc_plots/pdf/samtools-stats-dp.pdf",
                "multiqc/multiqc_plots/pdf/samtools_alignment_plot-cnt.pdf",
                "multiqc/multiqc_plots/pdf/samtools_alignment_plot-pct.pdf",
                "multiqc/multiqc_plots/pdf/vcftools_tstv_by_count.pdf",
                "multiqc/multiqc_plots/pdf/vcftools_tstv_by_qual.pdf",
                "multiqc/multiqc_plots/png",
                "multiqc/multiqc_plots/png/bcftools_stats_indel-lengths.png",
                "multiqc/multiqc_plots/png/bcftools_stats_vqc_Count_Indels.png",
                "multiqc/multiqc_plots/png/bcftools_stats_vqc_Count_SNP.png",
                "multiqc/multiqc_plots/png/bcftools_stats_vqc_Count_Transitions.png",
                "multiqc/multiqc_plots/png/bcftools_stats_vqc_Count_Transversions.png",
                "multiqc/multiqc_plots/png/fastqc-status-check-heatmap.png",
                "multiqc/multiqc_plots/png/fastqc_adapter_content_plot.png",
                "multiqc/multiqc_plots/png/fastqc_per_base_n_content_plot.png",
                "multiqc/multiqc_plots/png/fastqc_per_base_sequence_quality_plot.png",
                "multiqc/multiqc_plots/png/fastqc_per_sequence_gc_content_plot_Counts.png",
                "multiqc/multiqc_plots/png/fastqc_per_sequence_gc_content_plot_Percentages.png",
                "multiqc/multiqc_plots/png/fastqc_per_sequence_quality_scores_plot.png",
                "multiqc/multiqc_plots/png/fastqc_sequence_counts_plot-cnt.png",
                "multiqc/multiqc_plots/png/fastqc_sequence_counts_plot-pct.png",
                "multiqc/multiqc_plots/png/fastqc_sequence_duplication_levels_plot.png",
                "multiqc/multiqc_plots/png/fastqc_sequence_length_distribution_plot.png",
                "multiqc/multiqc_plots/png/fastqc_top_overrepresented_sequences_table.png",
                "multiqc/multiqc_plots/png/gatk-base-recalibrator-quality-scores-plot_Pre-recalibration_Count.png",
                "multiqc/multiqc_plots/png/gatk-base-recalibrator-quality-scores-plot_Pre-recalibration_Percent.png",
                "multiqc/multiqc_plots/png/gatk-base-recalibrator-reported-empirical-plot.png",
                "multiqc/multiqc_plots/png/general_stats_table.png",
                "multiqc/multiqc_plots/png/mosdepth-coverage-per-contig-single-cnt.png",
                "multiqc/multiqc_plots/png/mosdepth-coverage-per-contig-single-pct.png",
                "multiqc/multiqc_plots/png/mosdepth-cumcoverage-dist-id.png",
                "multiqc/multiqc_plots/png/picard_deduplication-cnt.png",
                "multiqc/multiqc_plots/png/picard_deduplication-pct.png",
                "multiqc/multiqc_plots/png/samtools-stats-dp.png",
                "multiqc/multiqc_plots/png/samtools_alignment_plot-cnt.png",
                "multiqc/multiqc_plots/png/samtools_alignment_plot-pct.png",
                "multiqc/multiqc_plots/png/vcftools_tstv_by_count.png",
                "multiqc/multiqc_plots/png/vcftools_tstv_by_qual.png",
                "multiqc/multiqc_plots/svg",
                "multiqc/multiqc_plots/svg/bcftools_stats_indel-lengths.svg",
                "multiqc/multiqc_plots/svg/bcftools_stats_vqc_Count_Indels.svg",
                "multiqc/multiqc_plots/svg/bcftools_stats_vqc_Count_SNP.svg",
                "multiqc/multiqc_plots/svg/bcftools_stats_vqc_Count_Transitions.svg",
                "multiqc/multiqc_plots/svg/bcftools_stats_vqc_Count_Transversions.svg",
                "multiqc/multiqc_plots/svg/fastqc-status-check-heatmap.svg",
                "multiqc/multiqc_plots/svg/fastqc_adapter_content_plot.svg",
                "multiqc/multiqc_plots/svg/fastqc_per_base_n_content_plot.svg",
                "multiqc/multiqc_plots/svg/fastqc_per_base_sequence_quality_plot.svg",
                "multiqc/multiqc_plots/svg/fastqc_per_sequence_gc_content_plot_Counts.svg",
                "multiqc/multiqc_plots/svg/fastqc_per_sequence_gc_content_plot_Percentages.svg",
                "multiqc/multiqc_plots/svg/fastqc_per_sequence_quality_scores_plot.svg",
                "multiqc/multiqc_plots/svg/fastqc_sequence_counts_plot-cnt.svg",
                "multiqc/multiqc_plots/svg/fastqc_sequence_counts_plot-pct.svg",
                "multiqc/multiqc_plots/svg/fastqc_sequence_duplication_levels_plot.svg",
                "multiqc/multiqc_plots/svg/fastqc_sequence_length_distribution_plot.svg",
                "multiqc/multiqc_plots/svg/fastqc_top_overrepresented_sequences_table.svg",
                "multiqc/multiqc_plots/svg/gatk-base-recalibrator-quality-scores-plot_Pre-recalibration_Count.svg",
                "multiqc/multiqc_plots/svg/gatk-base-recalibrator-quality-scores-plot_Pre-recalibration_Percent.svg",
                "multiqc/multiqc_plots/svg/gatk-base-recalibrator-reported-empirical-plot.svg",
                "multiqc/multiqc_plots/svg/general_stats_table.svg",
                "multiqc/multiqc_plots/svg/mosdepth-coverage-per-contig-single-cnt.svg",
                "multiqc/multiqc_plots/svg/mosdepth-coverage-per-contig-single-pct.svg",
                "multiqc/multiqc_plots/svg/mosdepth-cumcoverage-dist-id.svg",
                "multiqc/multiqc_plots/svg/picard_deduplication-cnt.svg",
                "multiqc/multiqc_plots/svg/picard_deduplication-pct.svg",
                "multiqc/multiqc_plots/svg/samtools-stats-dp.svg",
                "multiqc/multiqc_plots/svg/samtools_alignment_plot-cnt.svg",
                "multiqc/multiqc_plots/svg/samtools_alignment_plot-pct.svg",
                "multiqc/multiqc_plots/svg/vcftools_tstv_by_count.svg",
                "multiqc/multiqc_plots/svg/vcftools_tstv_by_qual.svg",
                "multiqc/multiqc_report.html",
                "pipeline_info",
                "pipeline_info/nf_core_sarek_software_mqc_versions.yml",
                "preprocessing",
                "preprocessing/mapped",
                "preprocessing/mapped/test",
                "preprocessing/mapped/test/test.sorted.bam",
                "preprocessing/mapped/test/test.sorted.bam.bai",
                "preprocessing/markduplicates",
                "preprocessing/markduplicates/test",
                "preprocessing/markduplicates/test/test.md.bam",
                "preprocessing/markduplicates/test/test.md.bam.bai",
                "preprocessing/recal_table",
                "preprocessing/recal_table/test",
                "preprocessing/recal_table/test/test.recal.table",
                "preprocessing/recalibrated",
                "preprocessing/recalibrated/test",
                "preprocessing/recalibrated/test/test.recal.bam",
                "preprocessing/recalibrated/test/test.recal.bam.bai",
                "reference",
                "reports",
                "reports/bcftools",
                "reports/bcftools/strelka",
                "reports/bcftools/strelka/test",
                "reports/bcftools/strelka/test/test.strelka.variants.bcftools_stats.txt",
                "reports/fastqc",
                "reports/fastqc/test-test_L1",
                "reports/fastqc/test-test_L1/test-test_L1_1_fastqc.html",
                "reports/fastqc/test-test_L1/test-test_L1_1_fastqc.zip",
                "reports/fastqc/test-test_L1/test-test_L1_2_fastqc.html",
                "reports/fastqc/test-test_L1/test-test_L1_2_fastqc.zip",
                "reports/fastqc/test-test_L2",
                "reports/fastqc/test-test_L2/test-test_L2_1_fastqc.html",
                "reports/fastqc/test-test_L2/test-test_L2_1_fastqc.zip",
                "reports/fastqc/test-test_L2/test-test_L2_2_fastqc.html",
                "reports/fastqc/test-test_L2/test-test_L2_2_fastqc.zip",
                "reports/markduplicates",
                "reports/markduplicates/test",
                "reports/markduplicates/test/test.md.cram.metrics",
                "reports/mosdepth",
                "reports/mosdepth/test",
                "reports/mosdepth/test/test.md.mosdepth.global.dist.txt",
                "reports/mosdepth/test/test.md.mosdepth.region.dist.txt",
                "reports/mosdepth/test/test.md.mosdepth.summary.txt",
                "reports/mosdepth/test/test.md.regions.bed.gz",
                "reports/mosdepth/test/test.md.regions.bed.gz.csi",
                "reports/mosdepth/test/test.recal.mosdepth.global.dist.txt",
                "reports/mosdepth/test/test.recal.mosdepth.region.dist.txt",
                "reports/mosdepth/test/test.recal.mosdepth.summary.txt",
                "reports/mosdepth/test/test.recal.regions.bed.gz",
                "reports/mosdepth/test/test.recal.regions.bed.gz.csi",
                "reports/samtools",
                "reports/samtools/test",
                "reports/samtools/test/test.md.cram.stats",
                "reports/samtools/test/test.recal.cram.stats",
                "reports/vcftools",
                "reports/vcftools/strelka",
                "reports/vcftools/strelka/test",
                "reports/vcftools/strelka/test/test.strelka.variants.FILTER.summary",
                "reports/vcftools/strelka/test/test.strelka.variants.TsTv.count",
                "reports/vcftools/strelka/test/test.strelka.variants.TsTv.qual",
                "variant_calling",
                "variant_calling/strelka",
                "variant_calling/strelka/test",
                "variant_calling/strelka/test/test.strelka.genome.vcf.gz",
                "variant_calling/strelka/test/test.strelka.genome.vcf.gz.tbi",
                "variant_calling/strelka/test/test.strelka.variants.vcf.gz",
                "variant_calling/strelka/test/test.strelka.variants.vcf.gz.tbi"
            ],
            [
                "bcftools_stats_indel-lengths.txt:md5,deccb75341ca46a6f09658f7fd9e348b",
                "bcftools_stats_vqc_Count_Indels.txt:md5,7b2a64880b653ccf0400ed9073e290dd",
                "bcftools_stats_vqc_Count_SNP.txt:md5,72e934e0e8ed9b9712105bbd66dd9ffd",
                "bcftools_stats_vqc_Count_Transitions.txt:md5,72e934e0e8ed9b9712105bbd66dd9ffd",
                "bcftools_stats_vqc_Count_Transversions.txt:md5,72e934e0e8ed9b9712105bbd66dd9ffd",
                "fastqc-status-check-heatmap.txt:md5,a020b9689ddeb4abec16b4854fe452f1",
                "fastqc_adapter_content_plot.txt:md5,2e1b72be741319e7fadbbb39d7e5b37d",
                "fastqc_per_base_n_content_plot.txt:md5,ad3b971a6bb4e8ba6c844c8a03584eb8",
                "fastqc_per_base_sequence_quality_plot.txt:md5,1bc03889d243a944253ac637d81ae10c",
                "fastqc_per_sequence_gc_content_plot_Counts.txt:md5,2c42d140ce06c08dad2b58f397c23239",
                "fastqc_per_sequence_gc_content_plot_Percentages.txt:md5,59e22821d350bfb97c37ffd9088f5ad9",
                "fastqc_per_sequence_quality_scores_plot.txt:md5,f33615cc98bb6225f39545a415fa7c0f",
                "fastqc_sequence_counts_plot.txt:md5,7f0f19a58e8e54e792a751fd04a9ae13",
                "fastqc_sequence_duplication_levels_plot.txt:md5,92b02e250ff78725deb9a10d510fcecc",
                "fastqc_sequence_length_distribution_plot.txt:md5,fb04dce68ec566314125bc9438211b28",
                "fastqc_top_overrepresented_sequences_table.txt:md5,d41d8cd98f00b204e9800998ecf8427e",
                "gatk-base-recalibrator-quality-scores-plot_Pre-recalibration_Count.txt:md5,20b2630a7400c9c279bf8c0c66341f7d",
                "gatk-base-recalibrator-quality-scores-plot_Pre-recalibration_Percent.txt:md5,b9b943621e50c7f3e75a37871667d5ed",
                "mosdepth-coverage-per-contig-single.txt:md5,264db67a99d2c90ea7b075e33c201b77",
                "mosdepth-cumcoverage-dist-id.txt:md5,5235e965da7ebe3bfebb24ffa88defff",
                "mosdepth_cov_dist.txt:md5,8d0d7cb485a7bffb07da17b28f827120",
                "mosdepth_cumcov_dist.txt:md5,8d0d7cb485a7bffb07da17b28f827120",
                "mosdepth_perchrom.txt:md5,264db67a99d2c90ea7b075e33c201b77",
                "multiqc_bcftools_stats.txt:md5,103ba59d44fc60e9308e64bbd0d0e504",
                "multiqc_citations.txt:md5,ace4ca89138a5f1e2be289c157c00bd9",
                "multiqc_fastqc.txt:md5,bde0d0bffa62228b33fb68b7e25b6ff8",
                "multiqc_samtools_stats.txt:md5,0f1e4c6c497d9a952765f9f3068ea4b9",
                "picard_histogram.txt:md5,d41d8cd98f00b204e9800998ecf8427e",
                "picard_histogram_1.txt:md5,d41d8cd98f00b204e9800998ecf8427e",
                "picard_histogram_2.txt:md5,d41d8cd98f00b204e9800998ecf8427e",
                "samtools-stats-dp.txt:md5,c94f4d3ffa3f510552f90e173fdd9f9d",
                "samtools_alignment_plot.txt:md5,717f499a3543e7ee4c7a8454bf80aeca",
                "vcftools_tstv_by_count.txt:md5,50efc5214fe2c39f21efb66a710d2ed6",
                "test.strelka.variants.bcftools_stats.txt:md5,86bd4938eed920d36f3f5937102a2967",
                "test.md.mosdepth.global.dist.txt:md5,b61e1acee11a6ddf7ce3232a5948a6a0",
                "test.md.mosdepth.region.dist.txt:md5,1a382f98d488d2ae3df83a0d87caafc1",
                "test.md.mosdepth.summary.txt:md5,839108358878ada89e1eaddf6e0541ba",
                "test.md.regions.bed.gz:md5,6fdaec99e739dc0f47fe55dd64dfe93e",
                "test.md.regions.bed.gz.csi:md5,5f9c60279af78e3aeafc96a8c11fb35f",
                "test.recal.mosdepth.global.dist.txt:md5,b61e1acee11a6ddf7ce3232a5948a6a0",
                "test.recal.mosdepth.region.dist.txt:md5,1a382f98d488d2ae3df83a0d87caafc1",
                "test.recal.mosdepth.summary.txt:md5,839108358878ada89e1eaddf6e0541ba",
                "test.recal.regions.bed.gz:md5,6fdaec99e739dc0f47fe55dd64dfe93e",
                "test.recal.regions.bed.gz.csi:md5,5f9c60279af78e3aeafc96a8c11fb35f",
                "test.md.cram.stats:md5,7d19da3fc342afe0884c944f97a578b1",
                "test.recal.cram.stats:md5,820d123e746d1abdc90fd8710828082e",
                "test.strelka.variants.FILTER.summary:md5,ad417bc96d31223f61170987975d8128",
                "test.strelka.variants.TsTv.count:md5,fa27f678965b7cba6a92efcd039f802a"
            ],
            [

            ]
        ],
        "meta": {
            "nf-test": "0.9.0",
            "nextflow": "24.04.4"
        },
        "timestamp": "2024-11-09T21:51:21.998984"
    }
}
